api/types/selector.go

```diff
@@ -28,6 +28,18 @@ type Selector struct {
 	LabelSelector string `json:"labelSelector,omitempty" yaml:"labelSelector,omitempty"`
 }
 
+func (s *Selector) Copy() Selector {
```

Comment:
just
```
func (s *Selector) Copy() Selector { return *s }
```
Of course either version will panic if s is nil

Reply:
done